cinema/views.py

```diff
@@ -87,6 +123,8 @@
         )
     )
     serializer_class = MovieSessionSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
The `MovieSessionViewSet` uses `TokenAuthentication` and `IsAdminOrIfAuthenticatedReadOnly` permissions. Ensure these are appropriate for the operations allowed on movie sessions.